packages/lambdas/src/sqs-to-converter.ts

```diff
@@ -220,42 +229,49 @@ export async function handler(event: SQSEvent) {
         if (!converterUrl) throw new Error(`Missing converterUrl`);
         const unusedSegments = attrib.unusedSegments?.stringValue;
         const invalidAccess = attrib.invalidAccess?.stringValue;
-        const converterParams = { patientId, fileName: s3FileName, unusedSegments, invalidAccess };
+        const converterParams: FhirConverterParams = {
+          patientId,
+          fileName: s3FileName,
+          unusedSegments,
+          invalidAccess,
+        };
 
         const preConversionFilename = `${s3FileName}.pre-conversion.xml`;
         const conversionResultFilename = `${s3FileName}.from_converter.json`;
 
         log(
           `Calling converter on url ${converterUrl} with params ${JSON.stringify(converterParams)}`
         );
-        const convertPayloadToFHIR = () =>
-          executeWithNetworkRetries(
-            () =>
-              fhirConverter.post(converterUrl, payloadClean, {
-                params: converterParams,
-                headers: { "Content-Type": TXT_MIME_TYPE },
-              }),
-            {
-              // No retries on timeout b/c we want to re-enqueue instead of trying within the same lambda run,
-              // it could lead to timing out the lambda execution.
-              log,
-            }
-          );
+
+        const partitionedPayload = partitionPayload(payloadClean);
```

Comment:
ya, good point.. I havent come up with a good name for it, so, since in the code it's `payloadClean`, im attaching the `clean` suffix. But lmk if you have better ideas